packages/flutter/lib/src/material/tab_bar_theme.dart

```diff
@@ -0,0 +1,94 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+
+import 'tabs.dart';
+
+/// Applies a theme to descendant [TabBar] widgets.
+///
+/// A tab bar theme describes the colors of the label and the size/shape of the
+/// [TabBar.indicator].
+///
+/// Descendant widgets obtain the current theme's [TabBarTheme] object using
+/// [ThemeData.tabBarTheme] on the [ThemeData] given by calling [Theme.of].
+/// [ThemeData.tabBarTheme] can be customized by copying it (using
+/// [TabBarTheme.copyWith]).
+///
+/// See also:
+///
+///  * [TabBar], Material Design widget that displays a horizontal row of tabs.
```

Comment:
Probably isn't necessary to reiterate "Material Design" here. Just "a widget that ...".

Reply:
Done.